plugins/processors/starlark/testdata/json_nested.star

```diff
@@ -0,0 +1,42 @@
+"""
+This code assumes the value parser with data_type='string' is used
+in the input collecting the JSON data. The entire JSON obj/doc will
+be set to a Field named `value` with which this code will work.
+
+Example input: ```
```

Comment:
Has to be 
```suggestion
Example Input: ```
```

Reply:
Furthermore I'm not sure if the current implementation of the test can cope with multi-line input. AFAIK we read one input per line, so please remove the nice formatting and make it an ugly one-liner. Never thought I will say this ever... ;-)

Reply:
Last but not least the given input/output has to be in line-protocol format... Please fix those issues to keep the starlark test-suite working...